cmd/status_test.go

```diff
@@ -0,0 +1,30 @@
+package cmd
+
+import (
+	"bytes"
+	"github.com/cobbler/cobblerclient"
+	"github.com/spf13/cobra"
+	"testing"
+)
+
+func Test_StatusCmd(t *testing.T) {
+	// Arrange
+	cobra.OnInitialize(initConfig, setupLogger)
+	rootCmd := NewRootCmd()
+	rootCmd.SetArgs([]string{"--config", "../testing/.cobbler.yaml", "status"})
+	stdout := bytes.NewBufferString("")
+	stderr := bytes.NewBufferString("")
+	rootCmd.SetOut(stdout)
+	rootCmd.SetErr(stderr)
+	expectedResult := "ip             |target              |start            |state            \n"
```

Comment:
Is the expected result missing or is it supposed to be emtpy?

Reply:
As our integration tests don't have data, this is supposed to be empty. Adding dummy data is a complexity I don't want to attempt at the moment.